### DISCUSSION TITLE
SRT Support

### DISCUSSION THREAD

**Discussion by capears** [bluenviron/mediamtx#1309] [Questions]
I saw in one of the issues something about supporting SRT. Has that been implemented? Looking for a way to pull an SRT stream and redistribute as RTSP with KLV data intact.

**Comment by aler9** [comment]
Hello, SRT support is planned but there isn't a fixed delivery date since implementation is really complex and must be built from scratch. You can follow updates here #563

[a comment by an automated account is not shown]
